lib/debt_ceiling/audit.rb

```diff
@@ -0,0 +1,80 @@
+module DebtCeiling
+  class Audit
+    extend Forwardable
+    include CommonMethods
+    undef :+
```

Comment:
This is a little funny, and first time I've ever actually used undef in practice, but I think makes sense here so that the 2 or 3 other classes that use CommonMethods module get + in one place, but it's not non-sensically part of the API of Audit.  Curious for opinions, but tempted to write this up as a pattern of some kind if I find another place to use, though it's also a funny module in that it only has two methods, and is being used for only one of them here.